.eslintrc.json

```diff
@@ -117,7 +109,23 @@
     "no-useless-call": "error",
     "no-unused-expressions": "error",
     "no-sequences": "error",
-    "no-self-compare": "error"
+    "no-self-compare": "error",
+    "no-case-declarations": "off"
+  },
+  "globals": {
+    "Association": true,
```

Comment:
This is going to be an issue with linting, now eslint will treat these as global and wont fail if someone is using `Model` without importing it first.

We can have a second linting phase for just jsdocs, with its own jsdocs related config in separate file.

Reply:
Or I just change those to typedefs.

Reply:
So isn't this thread still valid. `npm run lint` wont catch misuse of these global?

Reply:
I forgot to push latest changes, unfortunately type references are broken: https://github.com/gajus/eslint-plugin-jsdoc/issues/109